packages/botonic-plugin-flow-builder/src/functions/conditional-queue-status.ts

```diff
@@ -26,5 +34,18 @@ export async function conditionalQueueStatus({
     }
   )
   const isAvailable = response.data.available
+
+  console.log('conditionalQueueStatus', { data: response.data })
+  const flowBuilderPlugin = getFlowBuilderPlugin(request.plugins)
+  if (flowBuilderPlugin.trackEvent) {
+    const eventName = 'QUEUE_STATUS'
+    const args = {
+      queue_id,
+      queue_name,
+      status: isAvailable ? 'open' : 'closed',
+    }
+    await flowBuilderPlugin.trackEvent(request, eventName, args)
```

Comment:
For next iterations, I would use the decorator pattern to automatically call this, but atm is quite overengineering. Let's if we need to extend this logic in the future